modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_table.scss

```diff
@@ -1,227 +1,34 @@
-.dnd-thead {
-	display: table-header-group;
-}
-
-.dnd-tbody {
-	display: table-row-group;
-
-	.dnd-tr {
-		&:hover,
-		&.selected {
-			background-color: $primary-l3;
-		}
-
-		&:nth-child(odd):not(:hover):not(.selected) {
-			background-color: $table-accent-bg;
-		}
-	}
-}
-
-.dnd-td,
-.dnd-th {
-	border-color: $table-list-border-color;
-	border-style: solid;
-	border-width: 0 1px 1px 0;
-	display: table-cell;
-	overflow: hidden;
-	padding: $table-cell-padding;
-	vertical-align: middle;
-	white-space: normal;
+.visualization-mode-table table {
+	table-layout: auto !important;
```

Comment:
Without this change there is a regression in setting of column widths. With fixed table layout, overriding widths with CSS like in `commerce-theme-minium` does not work.